.github/workflows/checks.yml

```diff
@@ -39,3 +39,8 @@ jobs:
       run: |
         pip install -U tox
         tox
+    - name: Twine check
+      run: |
+        pip install twine
+        python setup.py sdist
+        twine check dist/*
```

Comment:
Could you please move this check to tox.ini, similar to other checks?

Reply:
Currently this check would be run 5 times on CI, for flake8, typing, docs, etc. tox environments. We should make it a separate tox env, and run this env on CI, similar to how others are executed.

Reply:
Done